build/System.Memory.props

```diff
@@ -1,5 +1,5 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6'">
```

Comment:
Instead we should change it to `'$(TargetFramework)' == 'netstandard2.0'`. Otherwise, we will have to change it again when we add the `net7.0` TFM in the future.